models/classes/session/Business/Domain/SessionCookieAttribute.php

```diff
@@ -43,4 +42,16 @@ public function __toString(): string
     {
         return "$this->name=$this->value";
     }
-}
+
+    public function getName():string
+    {
+        return $this->name;
+    }
+
+
+    public function getValue()
+    {
+        return $this->value;
+    }
+
+}
```

Comment:
```suggestion
}

```
https://www.php-fig.org/psr/psr-12/#22-files